core/cmd/prompter.go

```diff
@@ -13,6 +13,8 @@ import (
 	"golang.org/x/term"
 )
 
+//go:generate mockery --name Prompter --output ./mocks/
```

Comment:
Ah yeah, `--case=underscore` here. (We should really add a mockery config file to the root)